server.py

```diff
@@ -75,6 +80,14 @@ def make_json_error(ex):
 app.register_blueprint(api, url_prefix='/api/v1')
 app.register_blueprint(ui, url_prefix='/ui')
 
+app.debug = True
```

Comment:
`app.debug` should probably be taken from settings. could be something like `app.debug = app.config['DEBUG']` i think